lib/ansible/template/__init__.py

```diff
@@ -20,29 +20,34 @@
 __metaclass__ = type
 
 import ast
-import contextlib
 import datetime
+import importlib
 import os
+import pkgutil
 import pwd
 import re
 import time
 
 from functools import wraps
-from io import StringIO
 from numbers import Number
 
 try:
     from hashlib import sha1
 except ImportError:
     from sha import sha as sha1
 
+try:
```

Comment:
Just import it from https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/common/_collections_compat.py